R/io.R

```diff
@@ -234,12 +235,15 @@ get_raster_val_matrix <- function(raster_object, polygon) {
   polys <- polygon %>%
     st_transform(crs = r_crs)
 
-  # calculate the number of unique land classes from the input raster that are in the target polygons
+  # calculate the frequency of unique land classes from the input raster that are in the target polygons
   n_lcs <- crop(raster_object, polys) %>%
     mask(polys) %>%
-    raster::getValues()
+    freq(useNA = "no") %>%
+    as.data.frame()
```

Comment:
Can we use `as_tibble` instead please, for consistency? The renaming can be done in the same pipe:

`... as_tibble() %>% rename(Group.1 = value, x = count)`